Dikke Ploaten/View/Extensions.swift

```diff
@@ -30,21 +30,22 @@ extension UIViewController {
 
 extension UITableView {
 	
-	func setEmptyMessage(_ message: String) {
-		let messageLabel = UILabel(frame: CGRect(x: 0, y: 0, width: self.bounds.size.width, height: self.bounds.size.height))
-		messageLabel.text = message
-		messageLabel.textColor = .black
-		messageLabel.numberOfLines = 0;
-		messageLabel.textAlignment = .center;
-		messageLabel.font = UIFont(name: "TrebuchetMS", size: 15)
-		messageLabel.sizeToFit()
-		
-		self.backgroundView = messageLabel;
-		self.separatorStyle = .none;
-	}
-	
-	func restore() {
-		self.backgroundView = nil
-		self.separatorStyle = .singleLine
+	func setEmptyMessage(_ message: String? = nil) {
+		if(message == nil){
```

Comment:
hier doe je best `guard let message = message else`, aangezien het een optional is, zou je altijd moeten streven om `if let` of `guard let` te gebruiken

Reply:
Vangt de if-else dit niet op?

Reply:
Ofja wat biedt if let/guard let extra?

Reply:
Ja maar het is niet mooi